lpa-submissions-e2e-tests/cypress/fixtures/anAppeal.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
stole an arbitrary appeal from team1's e2e tests; there would likely be an advantage to testing more than one appeal type at some stage...

Reply:
for consistency this the data that is seeded: https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/feature/as-1903-lpa-sessions/data/data/appeals-service-api/001-appeals.json

Reply:
in all honesty we should have a bunch of these that we feed in to different tests to triangulate with.. if you check the assertions somewhere down below against the 'appeal sidebar' we're checking that the value displayed is the value we put in, so if we can't just swap appeals in and out we're presumably finding bugs?